homeassistant/components/reolink/store.py

```diff
@@ -0,0 +1,45 @@
+"""Local storage for the Reolink integration."""
+
+import asyncio
+import logging
+from pathlib import Path
+
+from homeassistant.core import HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ReolinkStore:
+    """Local storage for Reolink."""
+
+    def __init__(self, hass: HomeAssistant, config_id: str) -> None:
+        """Initialize ReolinkStore."""
+        self._hass = hass
+        self._path = Path(hass.config.path(f".storage/reolink.{config_id}.json"))
```

Comment:
Can we store all reolink files in a subfolder? 
See https://github.com/home-assistant/core/pull/112047 as example
Please clean up the storage files on config entry removal similar to the mentioned PR

Reply:
Done, but still need to test and write tests

Reply:
tested and works, tests have also been written.